services/viva-ms/test/helpers/caseDecision.test.js

```diff
@@ -154,7 +154,29 @@ it('Before the Viva administrator processes the application the case status must
 });
 
 it(`Calculation exists in the Viva workflow collection, then status type must be ${ACTIVE_PROCESSING}`, () => {
-  const results = decideNewCaseStatus(workflowCalculation);
+  const workflow = {
```

Comment:
You have already declared `workflow` constant in a more global scope. Consider renaming this one to avoid conflicts.

And also, a little cleanup

```js
const yourConstantName = {
   ...workflow,
   islocked: '123',
  calculations: {
      calculation: {},
    },
}
```